frontend/src/components/Dashboard/Dashboard.tsx

```diff
@@ -53,68 +60,76 @@ const columns = [
 	}),
 ];
 
-const patient = {
-	imgSrc:
-		'https://images.unsplash.com/photo-1542740348-39501cd6e2b4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1587&q=80',
-	givenName: 'Emily',
-	surname: 'Cooper',
-};
+
+
+
 
 const Dashboard = () => {
 	// const dispatch = useAppDispatch();
-	// const doctor = useAppSelector(state => state.doctors);
+    // useEffect(() => {
```

Comment:
Are we still using the commented code at some point? @ThanhNguyenn

Reply:
Yes Eddie! The portion of code that you pointed out is the general command to get the data from redux store. It still is WIP as I'm trying to create the doctor table from doctor perspective the table data component will depend who logs into the application (Could be doctor/patient/clinic).